pwndbg/color.py

```diff
@@ -80,6 +114,57 @@ def rwx(x):
 def highlight(x):
     return generateColorFunction(pwndbg.config.color_highlight)(x)
 
+def register(x):
```

Comment:
Given the vast quantity of registers and the sub-heirarchies, (`register_xxx` and `nearpc_xxx`) it seems like this doesn't really scale particularly well.

In particular, this ends up with us needing to invoke functions like `pwndbg.color.nearpc_symbol`.

Consider instead in a future revision that we might do something like:

```py
import pwndbg.color.nearpc as N`

print(N.symbol('foobar'))
```

Having an actual object hierarchy rather than an underscore-separated hierarchy might let us get away with vaguely cleaner code.

In the meantime it's not important since we have a manageable number of color settings, but it's something worth considering.